2005-03-04  Eric B. Weddington  <ericw@evcohs.com>

	* gas/config/tc-avr.c:
	Add support for atmega325, atmega3250, atmega645, atmega6450.

--- gas/config/tc-avr.c.old	2005-03-04 09:52:14.321375000 -0700
+++ gas/config/tc-avr.c	2005-03-04 09:25:06.805750000 -0700
@@ -107,7 +107,11 @@
   {"atmega169", AVR_ISA_M323,     bfd_mach_avr5},
   {"atmega32",  AVR_ISA_M323,     bfd_mach_avr5},
   {"atmega323", AVR_ISA_M323,     bfd_mach_avr5},
+  {"atmega325", AVR_ISA_M323,     bfd_mach_avr5},
+  {"atmega3250",AVR_ISA_M323,     bfd_mach_avr5},
   {"atmega64",  AVR_ISA_M323,     bfd_mach_avr5},
+  {"atmega645", AVR_ISA_M323,     bfd_mach_avr5},
+  {"atmega6450",AVR_ISA_M323,     bfd_mach_avr5},
   {"atmega128", AVR_ISA_M128,     bfd_mach_avr5},
   {"at90can128",AVR_ISA_M128,     bfd_mach_avr5},
   {"at94k",     AVR_ISA_94K,      bfd_mach_avr5},
